src/drivers/driver.ts

```diff
@@ -175,14 +175,13 @@ export abstract class CMakeDriver implements vscode.Disposable {
   /**
    * Compute the environment variables that apply with substitutions by expansionOptions
    */
-  async computeExpandedEnvironment(in_env: proc.EnvironmentVariables, expanded_env: proc.EnvironmentVariables): Promise<proc.EnvironmentVariables> {
-    const env = {} as {[key: string]: string};
+  async computeExpandedEnvironment(toExpand: util.Dictionary<string>, expanded: proc.EnvironmentVariables): Promise<util.Dictionary<string>> {
+    const env: util.Dictionary<string> = {};
     const opts = this.expansionOptions;
 
-    await Promise.resolve(
-      util.objectPairs(in_env)
-      .forEach(async ([key, value]) => env[key] = await expand.expandString(value, {...opts, envOverride: expanded_env}))
```

Comment:
Stumbled on this... I think the way this was written was incorrect and won't be properly awaited.